src/internet_identity/tests/integration/delegation.rs

```diff
@@ -288,26 +288,24 @@ fn should_issue_different_principals() -> Result<(), CallError> {
     let env = env();
     let canister_id = install_ii_canister(&env, II_WASM.clone());
     let user_number = flows::register_anchor(&env, canister_id);
-    let frontend_hostname_1 = "https://dapp1.com";
-    let frontend_hostname_2 = "https://dapp2.com";
```

Comment:
nit: the reason for "in-lining" these is that they were used only once, right? 
I believe there is still some advantage of having named constants (`frontend_hostname_1` is more informative than `https://dapp1.com`) so maybe we could keep them anyway?